plugins/view-resources/src/components/ViewOptions.svelte

```diff
@@ -65,49 +65,62 @@
     dispatch('update', { key: model.key, value: viewOptions[model.key] })
   }
 
+  // checking if selector provides multiple choice options
+  const hasMultipleSelections = (varTocheck: any) => {
+    if (!varTocheck) return false
+    if (Array.isArray(varTocheck)) {
+      return varTocheck.filter((item) => item !== undefined && item !== null && item !== '').length > 1
```

Comment:
```suggestion
      return varTocheck.some((item) => item != undefined && item !== '')
```

Reply:
does not fit the purpose - checking if having **multiple** selections and not a single stub value